editor/debugger/script_editor_debugger.cpp

```diff
@@ -445,6 +449,9 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, uint64_t p_thread
 			EditorDebuggerRemoteObjects *objs = inspector->set_objects(p_data);
 			emit_signal(SNAME("remote_objects_updated"), objs);
 		}
+	} else if (p_msg == "scene:debug_mute_audio") {
+		ERR_FAIL_COND(p_data.is_empty());
+		// This is handled by SceneDebugger, we need to ignore here to not show a WARNING.
```

Comment:
No need to have "WARNING" in all caps, it just draws unnecessary attention. Otherwise, LGTM.